WordPress/gutenberg
# Font Library: refactor endpoint permissions

## What?
Font Library:
- Refactor endpoint permissions.
- Standardize the output of the install and uninstall endpoints.
- Display a success or error notice on the frontend for install and uninstall actions.

## Why?
- To have a standard output for the endpoints for success and error cases.

## How?
Refactoring the endpoint responses.

## Testing Instructions
- Use the font library to install local and Google fonts.
- Remove fonts
Check that the result is expected and you get the success or error notices.


Props to @pbking and @jffng for the big help implementing this :)


Closes: https://github.com/WordPress/gutenberg/issues/54751
